dev/buildtool/spin_commands.py

```diff
@@ -190,19 +190,24 @@ def __determine_internal_version(self, repository):
     gate_major = match.group(1)
     gate_min = match.group(2)
 
-    tag_matcher = re.compile(r'version-{maj}.{min}.(\d+)'
-                             .format(maj=gate_major, min=gate_min))
+    tag_pattern = r'version-{maj}.{min}.(\d+)'.format(maj=gate_major, min=gate_min)
```

Comment:
FYI you could replace #185 with SemVer.make(self.__gate_version)
https://github.com/spinnaker/spinnaker/blob/master/dev/buildtool/git_support.py#L169

Reply:
Switched over to the SemVer class, I had to formulate the raw gate versions as tags but other than that it worked swell.